src/main/pg/bus_spi.c

```diff
@@ -31,6 +31,12 @@
 
 #include "bus_spi.h"
 
+#ifndef SPI0_SCK_PIN
```

Comment:
(better be explicit about device_0 )
```suggestion
#ifdef USE_SPI_DEVICE_0
#ifndef SPI0_SCK_PIN
```